docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -1,272 +1,115 @@
-    }
-}
-----------------------------------------------------------
-// CONSOLE
-
-To take more of the {stack} features for a spin, you can start a 30-day trial
-from Kibana or with the Start Trial API:
-
-[source,json]
-----------------------------------------------------------
-POST _license/start_trial
-----------------------------------------------------------
-// CONSOLE
-
-The 30-day trial enables you to try out the full set of platinum features,
-including security, machine learning, alerting, graph capabilities, and more.
-
-[role="xpack"]
-[[xpack-stack-upgrade]]
-=== Upgrading from 5.6
+When you've made the necessary changes and are ready to upgrade from 6.7 to
+{version}:
 
-{xpack} 5.6 provides migration and upgrade APIs for Elasticsearch and a
-Upgrade Assistant UI for Kibana. These tools are included with the trial
-license and the free basic license.
-
-To upgrade to {version} from 5.6:
+. Test the upgrade in a dev environment *before* upgrading your
+production cluster.
 
-. {ref}/setup-upgrade.html[Upgrade Elasticsearch] to the most recent 5.6 and
-install {xpack} on all nodes in your cluster. If you are upgrading from an
-earlier 5.x release, you can perform a rolling upgrade. To upgrade from older
-versions you must perform a full cluster restart.
-+
-If your trial license expires,
-https://register.elastic.co/[register for a free Basic license]. To apply the
-license, upload the license file with the `license` API:
-+
-[source,json]
-----------------------------------------------------------
-license -d @license.json
-----------------------------------------------------------
+. Back up your data. You **cannot roll back** to an earlier version unless
+you have a snapshot of your data. For information about creating snapshots, see
+{ref}/modules-snapshots.html[Snapshot and Restore].
 
-. If {xpack} **IS NOT** normally a part of your {stack}, disable the
-{es} {security-features} in `elasticsearch.yml`:
-+
-[source,yaml]
-----------------------------------------------------------
-xpack.security.enabled: false
-----------------------------------------------------------
+. Consider closing {ml} jobs before you start the upgrade process. While {ml}
+jobs can continue to run during a rolling upgrade, it increases the overhead
+on the cluster during the upgrade process. For more information, see
+{ref}/rolling-upgrades.html[Rolling upgrades].
 
-. Upgrade {kib} to the most recent 5.6 and install {xpack}.
+. Upgrade the components of your Elastic Stack in the following order:
 
-. If you disabled the {es} {security-features}, also disable the {kib}
-{security-features} in `kibana.yml`:
-+
-[source,yaml]
-----------------------------------------------------------
-xpack.security.enabled: false
-----------------------------------------------------------
+.. {es} Hadoop: {hadoop-ref}/install.html[install instructions]
+.. {es}: {ref}/setup-upgrade.html[upgrade instructions]
+.. Kibana: {kibana-ref}/upgrade.html[upgrade instructions]
```

Comment:
Per feedback from @tylersmalley, this is a little more complicated if you're using Kibana. Because Kibana comes with a new Upgrade Assistant which tells you if there are problems with your indices and cluster, the order should be:

1. Once the user is on 6.7, go into Kibana and view the Upgrade Assistant. Address all critical items.
2. Follow the ES upgrade steps.
3. Follow the Kibana upgrade steps.

For context, here's what the Upgrade Assistant looks like (this screenshot may contain bugs so it's not appropriate for use in the docs):

![image](https://user-images.githubusercontent.com/1238659/54648957-6fd6a100-4a65-11e9-98c5-f2de2e3196e1.png)